.github/workflows/ci-test.yml

```diff
@@ -537,13 +533,10 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0 #get all the history!!!
       - uses: ./.github/actions/build-setup
-      - uses: actions/cache@v2.1.6
-        with:
-          path: "/opt/cargo/git\n/opt/cargo/registry\n/opt/cargo/.package-cache"
-          key: crates-${{ runner.os }}-${{ hashFiles('Cargo.lock') }}
-          restore-keys: "crates-${{ runner.os }}"
+      - uses: Swatinem/rust-cache@c5ed9ba6b7e1bb8aff90d43acd2f0af4990fa57c
```

Comment:
It looks like this actions produces a `cache-hit` output (https://github.com/Swatinem/rust-cache#outputs).  We can consider adding a step to print it for future debugging.

Reply:
Shouldn't need to print it, if you look at the step it'll clearly indicate if there was a cache hit or a miss. The output would be useful if we wanted to programmatically do something based on the cache hit/miss event